reference/7.1/Microsoft.PowerShell.Core/About/about_Parameters.md

```diff
@@ -21,11 +21,24 @@ parameters follow the command name and have the following form:
 -<parameter_name> <parameter_value>
 -<parameter_name>:<parameter_value>
 ```
-
-The name of the parameter is preceded by a hyphen (-), which signals to PowerShell that the word
-following the hyphen is a parameter name. The parameter name and value can be separated by a space
-or a colon character. Some parameters do not require or accept a parameter value. Other parameters
+<!--
+01234567890123456789012345678901234567890123456789012345678901234567890123456789
+-->
+The name of the parameter is preceded by a hyphen (-), which signals to
+PowerShell that the word following the hyphen is a parameter name.
+This effect can be switched off by escaping the hyphen (``Write-Output `-I``)
+or specifying the string as a parameter value (`Write-Output -InputObject:-I`).
+The special parameter `--` switches off recognising parameters for the rest
+of the command (`Write-Output -- -I`).
+The parameter name and value can be separated by a space or a colon character.
+Some parameters do not require or accept a parameter value. Other parameters
 require a value, but do not require the parameter name in the command.
+Named parameters passed to a command that does not expect them become positional
+parameters (strings) followed by their given values, if any.
+If a parameter name is abbreviated, like `-I`, PowerShell will pick the parameter
+that begins with this prefix, like `-InputObject`, provided that there is only
```

Comment:
can we end the sentence after `-InputObject`? These clasuses could be two separate sentences and runs on as it is right now. So, it could read something like this:

If a parameter name is abbreviated, like `-I`, PowerShell picks the parameter that begins with this prefix, like `-InputObject`. This only works if there is a single parameter starting with the specified prefix. Using this feature in scripts is not recommended as future versions of commands could include new parameters that start with the same prefix.